src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -535,20 +535,35 @@ private static function initPlaceholderFormatters(): array
                 return $display;
             },
             'elapsed' => fn (self $bar) => Helper::formatTime(time() - $bar->getStartTime()),
```

Comment:
Why not refine the existing data points and make them more exact instead of doubling?

Reply:
My idea was to keep the existing api and just add new parameters if it's needed that the times are a bit more finetuned.